.github/workflows/docs.yml

```
@@ -21,7 +21,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.9
+          python-version: 3.10
```

Maybe you need "3.10" here?

```suggestion
          python-version: "3.10"
```

---

yep looks like it's the issue here

https://github.com/actions/setup-python/issues/249#issuecomment-934299359

---

True, thanks :+1: